aws/ecs_cluster_test.go

```diff
@@ -11,18 +11,17 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-const region = "eu-west-1"
-
 // Test we can create a cluster, tag it, and then find the tag
 func TestCanTagEcsClusters(t *testing.T) {
 	t.Parallel()
+	region, err := getRandomRegion()
 
 	awsSession, err := session.NewSession(&awsgo.Config{
 		Region: awsgo.String(region),
 	})
 	require.NoError(t, err)
 
-	cluster := createEcsFargateCluster(t, awsSession, util.UniqueID())
+	cluster := createEcsFargateCluster(t, awsSession, "cloud-nuke-test-" + util.UniqueID())
```

Comment:
Sanity check: is Fargate supported in all AWS regions? If not, you might  have to reduce the list.

Reply:
very good point 👍 thanks